AirCasting/CoreData/MeasurementStreamStorage.swift

```diff
@@ -7,6 +7,7 @@ import Foundation
 import Combine
 
 protocol MeasurementStreamStorage {
+    func accessStorage(_ task: @escaping(MeasurementStreamStorage) -> Void)
```

Comment:
some PRs ago (when `save()` was introduced) I proposed a solution that would ensure secure access during compile time instead of runtime, maybe we can revisit